http/action/src/test/java/io/knotx/databridge/http/action/HttpActionTest.java

```diff
@@ -325,6 +326,116 @@ void errorTransitionWhenResponseIsNotJson(VertxTestContext testContext,
         testContext);
   }
 
+  @Test
+  @DisplayName("Expect response body for a raw response configured action")
+  void successTransitionGivenConfiguredActionWhenResponseIsNotJson(VertxTestContext testContext,
+      Vertx vertx) throws Throwable {
+    // given, when
+    String endpointPath = "/api/non-json";
+
+    String responseBody = "<html>Hello</html>";
+    wireMockServer.stubFor(get(urlEqualTo(endpointPath))
+        .willReturn(aResponse().withBody(responseBody)));
+
+    ClientRequest clientRequest = prepareClientRequest(MultiMap.caseInsensitiveMultiMap(),
+        MultiMap.caseInsensitiveMultiMap(), endpointPath);
+
+    EndpointOptions endpointOptions = new EndpointOptions()
+        .setPath(endpointPath)
+        .setDomain("localhost")
+        .setPort(wireMockServer.port())
+        .setAllowedRequestHeaderPatterns(Collections.singletonList(Pattern.compile(".*")));
+
+    HttpAction tested = new HttpAction(vertx,
+        new HttpActionOptions()
+            .setEndpointOptions(endpointOptions)
+            .setResponseType("raw"), ACTION_ALIAS);
+
+    // then
+    verifyExecution(tested, clientRequest, FRAGMENT,
+        fragmentResult -> {
+          assertEquals(SUCCESS_TRANSITION, fragmentResult.getTransition());
+          JsonObject result = fragmentResult.getFragment().getPayload().getJsonObject("_result");
+          assertNotNull(result);
+          assertEquals(new JsonObject().put("body", responseBody), result);
+        }, testContext);
+  }
+
+  @Test
+  @DisplayName("Expect response metadata transition for a raw response configured action")
+  void validMetadataGivenConfiguredActionWhenResponseHasMetadata(VertxTestContext testContext,
+      Vertx vertx) throws Throwable {
+    // given, when
+    String endpointPath = "/api/non-json";
+
+    String responseBody = "<html>Hello</html>";
```

Comment:
Can we please add some more variants, with some special chars etc. You can save all responses to files and use [JUnit 5 Parametrized Tests](https://www.baeldung.com/parameterized-tests-junit-5).